### ISSUE TITLE
Active/Hover state in Vertical Nav and Pagination

### ISSUE THREAD

**Issue by marcoscv-work** [liferay/clay#5490]
The current style of the vertical navigation has some contrast problems for the hover and active states, this document aims to fix those problems and provide a clear way to see which navigation item is selected/hover.

_Note_: The new style has to be the replacement for the old one being applied to DXP, to correct it in DXP and propagate the change.

**Vertical Nav changes:**
Mouse Hover / Active state:
<img alt="" width="500" src="https://user-images.githubusercontent.com/7963804/234810183-8ccde41c-c0fb-4da2-b20b-c8d995857dd8.png">

Focus in active and focus in inactive example:
<img alt="" width="500" src="https://user-images.githubusercontent.com/7963804/234082726-aefd2906-f0fe-4ed9-83fc-b56eaa254e2c.png">

**Pagination**:
Active/hover state examples:

<img alt="" width="400" src="https://user-images.githubusercontent.com/7963804/234810342-97fb4a0e-188a-4ca7-8873-a7bb44f649d9.png">

Full documentation:
https://liferay.atlassian.net/wiki/spaces/ENGLEXICON/pages/2337374209/Vertical+Nav+-+Review+for+accessibility

Related to LPS:
https://issues.liferay.com/browse/LPS-168530

/cc @emiliano-cicero

**Comment by ethib137** [comment]
@marcoscv-work & @emiliano-cicero this is looking good. I have one question though. Is it a problem that color is the only thing that clearly distinguishes between hover and active states. For example, look at the pagination with saturation removed. There is a slight difference in background opacity from 4% to 6%, but it's not much. Is this sufficient or do we need more to distinguish between these states? 

![PaginationBlack White](https://user-images.githubusercontent.com/1609196/234507111-8fe0fb7e-6928-442c-a37d-34e506a23139.png)

**Comment by marcoscv-work** [comment]
Make sense, @emiliano-cicero can we improve this hover in some way?

I think we can go ahead an implement the changes and then include the hover improvement

**Comment by emiliano-cicero** [comment]
Hey guys, I included a new variation for the hover, instead of 4px on hover we can use 2px so other than the color change there's also a size increase for the active state. 

I've already updated the [documentation on confluence](https://liferay.atlassian.net/wiki/spaces/ENGLEXICON/pages/2337374209/Vertical+Nav+-+Review+for+accessibility)

Thanks @ethib137 for bringing this up! :D

**Comment by ethib137** [comment]
Looks great! Thanks guys!
